packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1662,6 +1662,7 @@
   "error-returning-inquiry": "Error returning inquiry to the queue",
   "error-role-in-use": "Cannot delete role because it's in use",
   "error-role-name-required": "Role name is required",
+  "error-role-already-present":"Role is already present with this name",
```

Comment:
```suggestion
  "error-role-already-present":"A role with this name already exists",
```

Reply:
@KevLehman I have made the requested changes for this pull request on PR #21905 as this repo got deleted